### PR TITLE
BaSP-TG-44: Add/Edit Super Admins

### PR DESCRIPTION
**Pull request by HernanCazeneuve**

### CODE REVIEW

File: src/Components/SuperAdmin/index.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+      setRequestType('PUT');
+      fetchId(superAdminId);
+    } else {
+      setRequestType('POST');
+    }
+  }, []);
+
+  async function handleSubmit(e) {
+    e.preventDefault();
+    if (firstName === '' || lastName === '' || email === '' || password === '') {
```
Comment (Gearyandres94) [inline src/Components/SuperAdmin/index.js:43]: This could be deleted if you make the fields required in html.

**Review by Gearyandres94 — COMMENTED** [review]
a little thing to improve but it looks really great.

**Review by Gearyandres94 — APPROVED** [review]
Everything looks great!! Great job!!

**Review by CarVigil — CHANGES_REQUESTED** [review]

**Review by Gearyandres94 — APPROVED** [review]
Great work!!!!

**Review by CarVigil — APPROVED** [review]

**Review by HiginiaMed — APPROVED** [review]

**Pull request merged**